test/integration/BUILD

```diff
@@ -21,11 +21,11 @@ load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 load("@graknlabs_common//test/server:rules.bzl", "grakn_java_test")
 
 grakn_java_test(
-    "test-client-query",
-    "@graknlabs_grakn_core_artifact_mac//file",
-    "@graknlabs_grakn_core_artifact_linux//file",
-    "@graknlabs_grakn_core_artifact_windows//file",
+    name = "test-client-query",
```

Comment:
I think this may have been working, somehow, using positional parameters? But if so, it was definitely weird, using named parameters is definitely better in Bazel.